<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GfxPaint::ColourSlidersWidget</class>
 <widget class="QWidget" name="GfxPaint::ColourSlidersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="colourSpaceLabel">
     <property name="text">
      <string>&amp;Colour Space</string>
     </property>
     <property name="buddy">
      <cstring>colourSpaceComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="colourSpaceComboBox"/>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="alphaCheckBox">
     <property name="text">
      <string>&amp;Alpha</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="quantiseCheckBox">
     <property name="text">
      <string>&amp;Quantise</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QVBoxLayout" name="colourSliderLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
